IMAGE_UPLOAD_FIX.md

```
@@ -0,0 +1,203 @@
+  } else {
+    // Original behavior: just return URL
+    toast({ title: 'Image uploaded' })
+    onUploadComplete?.(imageUrl)
+  }
+}
+```
+
+## Database Flow
+
+### During Campaign Creation
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)